lite/examples/pose_estimation/android/app/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,20 @@
+<resources>
+    <string name="app_name">Movenet</string>
+    <string name="tfe_pn_request_permission">This app needs camera permission.</string>
+    <string name="tv_score">Score: %.2f</string>
```

Comment:
Please prefix: "tfe_pe_" for all strings, which is to differentiate for each examples. 
- pe: poseestimation